MAUI/Carousel-View/LoadMore.md

```diff
@@ -0,0 +1,517 @@
+            <carousel:SfCarousel.LoadMoreView>
+                <Grid BackgroundColor="#FFFFFFFF">
+                    <Label
+                        Text="Load More..."
+                        FontSize="14"
+                        TextColor="#FF000000"
+                        FontAttributes="Bold"
+                        HorizontalTextAlignment="Center"
+                        VerticalTextAlignment="Center"
+                        HorizontalOptions="Center"
+                        VerticalOptions="Center"/>
+                </Grid>
+            </carousel:SfCarousel.LoadMoreView>
+        </carousel:SfCarousel>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Carousel;
+using System.Collections.Generic;
+
+namespace CarouselSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            this.BindingContext = new CarouselViewModel();
+            SfCarousel carousel = new SfCarousel()
+            {
+                ItemHeight = 200,
+                ItemWidth = 200,
+                ItemSpacing = 2,
+                AllowLoadMore = true,
+                LoadMoreItemsCount = 2,
+                ViewMode = ViewMode.Linear
+            };
+
+            carousel.ItemTemplate = new DataTemplate(() =>
+            {
+                Image image = new Image();
+                image.SetBinding(Image.SourceProperty, "Image");
+                image.Aspect = Aspect.AspectFit;
+                return image;
+            });
+
+            carousel.SetBinding(SfCarousel.ItemsSourceProperty, "ImageCollection");
+
+            Grid grid = new Grid()
+            {
+                BackgroundColor = Color.White
+            };
+
+            Label label = new Label()
+            {
+                Text = "Load More...",
+                FontSize = 14,
+                TextColor = Color.Black,
+                FontAttributes = FontAttributes.Bold,
+                HorizontalOptions = LayoutOptions.Center,
+                VerticalOptions = LayoutOptions.Center,
+                HorizontalTextAlignment = TextAlignment.Center,
+                VerticalTextAlignment = TextAlignment.Center
+            };
+
+            grid.Children.Add(label);
+
+            carousel.LoadMoreView = grid;
+
+            this.Content = carousel;
+        }
+    }
+
+    public class CarouselModel
+    {
+        public CarouselModel(string imageString)
+        {
+            Image = imageString;
+        }
+        private string _image;
+
+        public string Image
+        {
+            get { return _image; }
+            set { _image = value; }
+        }
+    }
+
+    public class CarouselViewModel
+    {
+        public CarouselViewModel()
+        {
+            ImageCollection.Add(new CarouselModel("cart.png"));
+            ImageCollection.Add(new CarouselModel("cart.png"));
```

Comment:
Add different image name in Each ImageCollection

Reply:
Addressed the changes.